vendor/engines/secure_rooms/app/controllers/secure_rooms/card_numbers_controller.rb

```diff
@@ -0,0 +1,54 @@
+module SecureRooms
+
+  class CardNumbersController < ApplicationController
+
+    include TextHelpers::Translation
+
+    admin_tab :all
+    customer_tab :password
+
+    layout "two_column"
+
+    before_action :init_current_facility
+    before_action :authenticate_user!
+    before_action :load_and_authorize_user_with_card_number
+    before_action :check_acting_as
+
+    def initialize
+      @active_tab = "admin_users"
+      super
+    end
+
+    def edit; end
+
+    def update
+      if @user.update_attributes(update_user_params)
+        flash[:notice] = text("controllers.users.update.success")
```

Comment:
We possibly could override `translation_scope` to return `"controllers.users"`, then we can can use text_helpers a little more succinctly.